app/femr/data/DataModelMapper.java

```diff
@@ -363,6 +363,7 @@ public IPatient createPatient(int userID, String firstName, String lastName, Dat
         patient.setUserId(userID);
         patient.setFirstName(firstName);
         patient.setLastName(lastName);
+        patient.setPhoneNumber(phoneNumber);
```

Comment:
Phone_number currently shows null for me in the DB when no phone number is entered.  

The column generates at default and has default = null, so when it receives an empty string it automagically assumes null.

ALTER TABLE patients ADD COLUMN phone_number VARCHAR(20) NULL DEFAULT NULL AFTER last_name;